saleor/webhook/__init__.py

```diff
@@ -61,11 +63,11 @@ class WebhookEventType:
     ]
 
     PERMISSIONS = {
-        ORDER_CREATED: "order.manage_orders",
-        ORDER_FULLY_PAID: "order.manage_orders",
-        ORDER_UPDATED: "order.manage_orders",
-        ORDER_CANCELLED: "order.manage_orders",
-        ORDER_FULFILLED: "order.manage_orders",
-        CUSTOMER_CREATED: "account.manage_users",
-        PRODUCT_CREATED: "product.manage_products",
+        ORDER_CREATED: OrderPermissions.MANAGE_ORDERS,
```

Comment:
Assignig here `.value` probably will be better. `PERMISSIONS[event_name]` instead of `PERMISSIONS[event_name].value`

Reply:
cannot do that

webhook resolver depends on this permissions as Enums not strings
```
def resolve_sample_payload(info, event_name):
    service_account = info.context.service_account
    required_permission = WebhookEventType.PERMISSIONS.get(event_name)
    if service_account and service_account.has_perm(required_permission):
```